ci.sh

```diff
@@ -1,6 +1,7 @@
 #!/bin/bash
 
 set -ex -o pipefail
+export PYRIGHT_PYTHON_IGNORE_WARNINGS=1
```

Comment:
TBF this probably is better off in the `check.sh` file, probably?

Reply:
added a comment